content-scripts/src/modules/icons/index.js

```diff
@@ -16,31 +17,49 @@ const replaceImages = () => {
     const handleImage = (i) => {
         const icon = IMG_ICON_MAP[i.src.substring(i.src.lastIndexOf("/") + 1)];
 
-        if (icon === undefined)
-            return i.classList.remove("se-icon", "se-hidden-icon");
+        let span = null;
 
-        if (icon === "") {
-            i.classList.add("se-hidden-icon");
-            i.classList.remove("se-icon");
+        if (i.nextElementSibling?.matches(".se-icon"))
+            span = i.nextElementSibling;
+
+        if (icon === undefined) {
+            span?.remove();
+            i.classList.remove("se-hidden-icon");
             return;
         }
 
-        i.classList.remove("se-hidden-icon");
-        i.classList.add("se-icon");
+        i.classList.add("se-hidden-icon");
 
-        const span = document.createElement("span");
-        replaceWithKeepAttrs(i, span);
+        if (icon === "") {
+            span?.remove();
+            return;
+        }
+
+        if (!span) {
+            span = document.createElement("span");
+            i.insertAdjacentElement("afterend", span);
+            copyEvents(i, span);
+        }
 
         size = Math.max(Math.round(Math.max(i.width, i.height) / 24) * 24, 24);
 
+        copyAttrs(i, span);
         span.classList.add(`ri-${icon}-line`);
         span.style.fontSize = `${size}px`;
+        span.classList.add("se-icon");
+        span.classList.remove("se-hidden-icon");
     };
 
     new MutationObserver((ms) =>
         ms.forEach((m) => {
-            if (m.type == "attributes") handleImage(m.target);
-            else m.target.querySelectorAll("img").forEach(handleImage);
+            console.log(m);
```

Comment:
Maybe remove this `console.log` if it's not necessary

Reply:
You're right, forgot to remove that